File: Reqnroll/Chapter13/13.3-TheIntroduceAndReferPattern/Before/WIMP.IntroduceAndReferSample.App/Models/Order.cs
namespace WIMP.IntroduceAndReferSample.App.Models;

public class Order(int orderNo, string customerName, string pizzaName)
{
    public int OrderNo { get; set; } = orderNo;
    public string CustomerName { get; set; } = customerName;
    public string PizzaName { get; set; } = pizzaName;
    public OrderStatus Status { get; set; } = OrderStatus.New;
}

File: CucumberJVM/Chapter15/15.7-StubDependencyPattern/Before/src/main/java/com/wimp/app/models/Order.java
package com.wimp.app.models;

import java.time.LocalDateTime;

public class Order {
    private int orderNo;
    private String customerName;
    private LocalDateTime expectedDeliveryTime;
    private OrderStatus status;

    public int getOrderNo() {
        return orderNo;
    }

    public void setOrderNo(int orderNo) {
        this.orderNo = orderNo;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public LocalDateTime getExpectedDeliveryTime() {
        return expectedDeliveryTime;
    }

    public void setExpectedDeliveryTime(LocalDateTime expectedDeliveryTime) {
        this.expectedDeliveryTime = expectedDeliveryTime;
    }

    public OrderStatus getStatus() {
        return status;
    }

    public void setStatus(OrderStatus status) {
        this.status = status;
    }
}
